Release/src/streams/fileio_win32.cpp

```diff
@@ -496,7 +496,7 @@ size_t _read_file_async(_In_ streams::details::_file_info_impl* fInfo,
 
     size_t result = static_cast<size_t>(-1);
 
-    if (wrResult == TRUE)
+    if (wrResult != FALSE)
```

Comment:
I think a better fix for these would be to just remove the comparison; `if (wrResult)`